examples/ML_classifier_copies/ML_copies_original_models.py

```diff
@@ -1,17 +1,24 @@
+import catboost
```

Comment:
This needs to be added to the environment.yml.

Reply:
I've added catboost to the environment.